### DISCUSSION TITLE
select version for documentation

### DISCUSSION THREAD

**Discussion by anasrar** [mantinedev/mantine#1938] [Q&A] [answered]
current documentation page was the latest release, can we have documentation base on version.
really useful for long term project that don't really need upgrade but still need read the documentation.

**Comment by rtivital** [comment, marked as answer]
You can find previous versions of documentation here https://mantine.dev/pages/about/#previous-versions-documentation

**Reply by anasrar** [reply]
i would love to make it more accessible, like dropdown menu in navbar, anyway thanks
